packages/elements-dev-portal/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-dev-portal",
-  "version": "3.0.7",
+  "version": "3.0.8",
```

IMO, we should stay consistent with the versioning and simply do this for all packages the same.

```suggestion
  "version": "3.0.701",
```